readme.md

```diff
@@ -30,16 +30,27 @@ The output is a JSON object in the form:
 
 ```json
 {
-    "bundle_name": {
-        "asset_kind": "/public/path/to/asset"
-    }
+    "chunks": {
+      "bundle_name": {
+          "asset_kind": "/public/path/to/asset"
+      }
+    },
+    "images": [{
+        "name": "hello.svg",
+        "path": "/public/path/to/asset/hello.svg"
+      }
+    ]
```

Comment:
@dimchez I want to make it a bit more generic, so it would be possible attach any kind of assets. It's reasonable to rename `images` to `assets`. WDYT?